src/main/java/flaxbeard/immersivepetroleum/common/util/commands/IslandCommand.java

```diff
@@ -22,29 +22,19 @@ private IslandCommand(){
 	}
 	
```

Comment:
Wouldnt be suprised if when i merge this i loose complete sense of what the fuck is even going on Lol

Yep, you guessed it: Readablity.